src/Parallel/PhaseControl/CheckpointAndExitAfterWallclock.hpp

```diff
@@ -77,6 +77,9 @@ struct CheckpointAndExitRequested {
  * \brief Phase control object that runs the WriteCheckpoint and Exit phases
  * after a specified amount of wallclock time has elapsed.
  *
+ * When the executable exits from here, it does so with
+ * `Parallel::ExitCode::ContinueFromCheckpoint`.
```

Comment:
ah, I didn't remember we had defined our own enum for exit codes.  Maybe mention that in the script when it talks about exit codes